lib/class-wp-theme-json-resolver.php

```diff
@@ -57,7 +57,7 @@ class WP_Theme_JSON_Resolver {
 	 *
 	 * @return array Contents that adhere to the theme.json schema.
 	 */
-	private static function read_json_file( $file_path ) {
+	public static function read_json_file( $file_path ) {
```

Comment:
I'm not sure this change is ok. It makes things easier for child themes, as they can just call this function directly, but if this change is not OK we can do without it.